src/measure_theory/function/l1_space.lean

```diff
@@ -446,6 +446,25 @@ end
 lemma integrable_const [is_finite_measure μ] (c : β) : integrable (λ x : α, c) μ :=
 integrable_const_iff.2 $ or.inr $ measure_lt_top _ _
 
+lemma mem_ℒp.integrable_norm_rpow {f : α → β} {p : ℝ≥0∞}
+  (hf : mem_ℒp f p μ) (hp_ne_zero : p ≠ 0) (hp_ne_top : p ≠ ∞) :
```

Comment:
Rather than having `p ≠ ∞`, would it be easier to change the type to `p : ℝ≥0`?

Reply:
No, because in most applications it will be to a `p` which is an `ennreal`, not the coercion of an `nnreal`. Essentially all lemmas for `L^p` spaces are phrased like that because of this.